packages/utils/upgrade/README.md

```diff
@@ -1 +1,179 @@
-<h1 align="center">upgrade</h1>
+# Strapi Upgrade Tool
+
+## Description
+
+The Strapi Upgrade Tool is the CLI for facilitating upgrades between Strapi versions.
+
+When run, the command will "step" from your current version of Strapi to the next version that includes code tranforms for breaking changes.
+
+From now on, all breaking changes, and ideally also deprecations, must have an accompanying code transform written to be accepted.
+
+### Types of Transforms
+
+The upgrade tool provides two types of transforms:
+
+- `json`: for updating a project's json files, primarily intended for the `package.json`
+- `code`: codemods; for updating a project's [js, jsx, ts, tsx] files
+
+### What's a codemod?
+
+Codemods are a scripted way to refactor code. Here we are providing and running these scripts for users for any changes necessary in user code between Strapi versions.
+
+For example, if we need to rename a package used by Strapi projects, instead of instructing users to change the package, we provide a script that searches through the user's project and does the replacement for them.
```

Comment:
```suggestion
For example, if we need to rename a package used by Strapi projects, instead of instructing users to change the import, we provide a script that searches through the user's project and does the replacement for them.
```